src/Raven.Client/Documents/Queries/IndexPatchOptions.cs

```diff
@@ -7,10 +7,21 @@ namespace Raven.Client.Documents.Queries
     /// </summary>
     public sealed class IndexPatchOptions
     {
+        public IndexPatchOptions()
+        {
+        }
+
+        public IndexPatchOptions(TimeSpan waitForIndexesTimeout)
+        {
+            WaitForIndexesTimeout = waitForIndexesTimeout;
+        }
+
+      
+
         /// <summary>
         /// Maximum time to wait for the indexes to become non-stale.
         /// </summary>
-        public TimeSpan? WaitForIndexesTimeout { get; set; }
+        public TimeSpan WaitForIndexesTimeout { get; set; }
```

Comment:
what I meant is to introduce (unless we have it) in DocumentConventions a static:
```
internal readonly static TimeSpan DefaultWaitForNonStaleResultsTimeout = TimeSpan.FromSeconds(15);
```

then use it in conventions to set the default one and also use it here:

```
public TimeSpan WaitForIndexesTimeout { get; set; } = DocumentConventions.DefaultWaitForNonStaleResultsTimeout;
```